charts/argo-cd/templates/argocd-application-controller/statefulset.yaml

```diff
@@ -142,8 +144,8 @@ spec:
           matchLabels:
             {{- include "argo-cd.selectorLabels" (dict "context" $ "name" $.Values.controller.name) | nindent 12 }}
         {{- end }}
+        {{- end }}
```

Comment:
revert the change (2 less spaces) or add 2 spaces on line 140 (`{{- range $constraint := . }}`)